help/include/sidebar_index.md

```diff
@@ -61,7 +61,7 @@
 * [Mention a user or group](/help/mention-a-user-or-group)
 * [Link to a channel, topic, or message](/help/link-to-a-message-or-conversation)
 * [Format a quote](/help/format-a-quote)
-* [Quote and reply](/help/quote-and-reply)
+* [Quote or forward a message](/help/quote-or-forward-a-message)
 * [Emoji and emoticons](/help/emoji-and-emoticons)
```

Comment:
Just to check, did you add a redirect for this page?

Reply:
Yes, I hope this is correct:
![image](https://github.com/user-attachments/assets/1cb3593b-69ce-4915-9c2b-723d2d293094)